### PR TITLE
Refactor(web, web-twig, web-react, form-validations): ID naming conventions across packages

### PR DESCRIPTION
**Pull request by pavelklibani**
## Description

- changed all IDs and attributes linked to those IDs to kebab-case

### Additional context

... few files were changed...

### Issue reference

[Sjednotit zápis id na kebab-case](https://jira.almacareer.tech/browse/DS-1097)

### CODE REVIEW

**Review by curdaj — APPROVED** [review]

**Comment by pavelklibani** [comment]
rebased after 🍌 release

File: packages/web-react/src/components/Accordion/demo/Accordion.tsx [hunk 1]
```diff
@@ -30,15 +30,15 @@ export const content = (
 );
 
 const Story = () => {
-  const [openState, setOpenState] = React.useState<AccordionOpenStateType>('AccordionItemExample1');
+  const [openState, setOpenState] = React.useState<AccordionOpenStateType>('AccordionItemExample-1');
```
Comment (literat) [inline packages/web-react/src/components/Accordion/demo/Accordion.tsx:33]:
```suggestion
  const [openState, setOpenState] = React.useState<AccordionOpenStateType>('AccordionItemExample-1');
```
```suggestion
  const [openState, setOpenState] = useState<AccordionOpenStateType>('AccordionItemExample-1');
```

**Review by literat — APPROVED** [review]
LGTM 👍 Please, just fix one suggestion. :-)

File: packages/web-react/src/components/Accordion/demo/Accordion.tsx (hunk 1 above)
Comment (pavelklibani) [reply packages/web-react/src/components/Accordion/demo/Accordion.tsx]: fixed, thank you

**Review by crishpeen — COMMENTED** [review]
So, no *lint rule?

File: packages/web-react/src/components/Accordion/demo/Accordion.tsx (hunk 1 above)
Comment (crishpeen) [reply packages/web-react/src/components/Accordion/demo/Accordion.tsx]: We should unify this everywhere but in another PR.

**Pull request merged**